janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchSetup.java

```
@@ -198,21 +208,21 @@ public Connection connect(Configuration config) throws IOException {
             log.debug("Set {}: {}", k, ignoreClusterName);
         }
 
-        // Force-enable dynamic scripting.  This is probably only useful in Node mode.
-        String disableScriptsKey = "script.disable_dynamic";
-        String disableScriptsVal = settings.get(disableScriptsKey);
-        if (null != disableScriptsVal && !"false".equals(disableScriptsVal)) {
-            log.warn("JanusGraph requires Elasticsearch dynamic scripting.  Setting {} to false.  " +
-                    "Dynamic scripting must be allowed in the Elasticsearch cluster configuration.",
-                    disableScriptsKey);
+        // Force-enable inline scripting.  This is probably only useful in Node mode.
+        String inlineScriptsKey = "script.inline";
+        String inlineScriptsVal = settings.get(inlineScriptsKey);
+        if (null != inlineScriptsVal && !"true".equals(inlineScriptsVal)) {
+            log.warn("JanusGraph requires Elasticsearch inline scripting.  Setting {} to true.  " +
```

requirement sounds like a user error. loglevel=error and fail fast?